### ISSUE TITLE
Exporting the operator _thnn_fused_lstm_cell to ONNX opset version 12 is not supported. 

### ISSUE THREAD

**Issue by hengyeliu** [pytorch/pytorch#47044]
## 🐛 Bug
While convert an RNN to onnx, it report this error. 
![屏幕截图(4)](https://user-images.githubusercontent.com/37336300/97544558-c14a5e80-1a04-11eb-81f3-cf6fbac242c6.png)
## To Reproduce

1.While run this code on GPU this error will be reported.
2.But if i run this code on CPU there is no error reported
`
    import torch
    import torch.nn as nn

    class StackedLSTM(nn.Module):
        def __init__(self, num_layers, input_size, rnn_size, dropout):
            super(StackedLSTM, self).__init__()
            self.dropout = nn.Dropout(dropout)
            self.num_layers = num_layers
            self.layers = nn.ModuleList()

            for _ in range(num_layers):
                self.layers.append(nn.LSTMCell(input_size, rnn_size))
                input_size = rnn_size

        def forward(self, input_feed, hidden):
            h_0, c_0 = hidden
            h_1, c_1 = [], []
            for i, layer in enumerate(self.layers):
                h_1_i, c_1_i = layer(input_feed, (h_0[i], c_0[i]))
                input_feed = h_1_i
                if i + 1 != self.num_layers:
                    input_feed = self.dropout(input_feed)
                h_1 += [h_1_i]
                c_1 += [c_1_i]
            h_1 = torch.stack(h_1)
            c_1 = torch.stack(c_1)
            return input_feed, (h_1, c_1)

    lstm = StackedLSTM(2, 580, 500, 0.3).cuda()
    h_t = torch.randn(2, 10, 500).cuda()
    h_c = torch.randn(2, 10, 500).cuda()
    a = torch.randn(10, 580).cuda()
    torch.onnx.export(lstm, (a, (h_t, h_c)), f='LSTM.onnx', opset_version=12, input_names=['input', 'h_t', 'h_c'], output_names= 
        ['output', 'hidden_t', 'hidden_c'])
`
## Expected behavior
Convert this RNN to onnx while runing on GPU.
## Environment
PyTorch version: 1.7.0
Is debug build: True
CUDA used to build PyTorch: 10.2
ROCM used to build PyTorch: N/A

OS: Microsoft Windows 10 专业版
GCC version: (GCC) 9.2.0
Clang version: Could not collect
CMake version: version 3.16.2

Python version: 3.6 (64-bit runtime)
Is CUDA available: True
CUDA runtime version: 10.2.89
GPU models and configuration: GPU 0: GeForce GTX 860M
Nvidia driver version: 441.22
cuDNN version: C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.2\bin\cudnn64_7.dll
HIP runtime version: N/A
MIOpen runtime version: N/A

Versions of relevant libraries:
[pip3] numpy==1.19.2
[pip3] numpydoc==0.8.0
[pip3] torch==1.7.0
[pip3] torchaudio==0.7.0
[pip3] torchtext==0.6.0
[pip3] torchvision==0.8.1
[conda] blas                      1.0                         mkl
[conda] cpuonly                   1.0                           0    pytorch
[conda] mkl                       2018.0.2                      1
[conda] mkl-service               1.1.2            py36h57e144c_4
[conda] mkl_fft                   1.0.1            py36h452e1ab_0
[conda] mkl_random                1.0.1            py36h9258bd6_0
[conda] numpy                     1.19.2                   pypi_0    pypi
[conda] numpydoc                  0.8.0                    py36_0
[conda] torch                     1.7.0                    pypi_0    pypi
[conda] torchaudio                0.7.0                    pypi_0    pypi
[conda] torchtext                 0.6.0                    pypi_0    pypi
[conda] torchvision               0.8.1                    pypi_0    pypi

cc @houseroad @spandantiwari @lara-hdr @BowenBao @neginraoof

**Comment by spandantiwari** [comment]
This is a duplicate of https://github.com/pytorch/pytorch/issues/25533. Closing this one, and we will track that issue. Please reopen if this is not a duplicate.
